config/environments/development.rb

```diff
@@ -26,22 +16,10 @@
     config.cache_store = :null_store
   end
 
-  # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
```

Comment:
group it, please, by purpose. 
E.g. configuration for action_mailer, active_support, etc